### DISCUSSION TITLE
pyenv installation fails on centos 7.x

### DISCUSSION THREAD

**Discussion by jitendranair** [pyenv/pyenv#2542] [General]
[j@l ~]$ curl https://pyenv.run | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   270  100   270    0     0     22      0  0:00:12  0:00:12 --:--:--    57
curl: (7) Failed connect to raw.githubusercontent.com:443; Operation now in progress

**Comment by native-api** [comment]
Not related to Pyenv. Googling the message shows that it can occur if you fail to connect via a proxy.

**Reply by jitendranair** [reply]
Right, it is not related to Pyenv. I changed the DNS server to 8.8.8.8 and it worked. 
Solution reference: https://stackoverflow.com/a/62198387/2843837. 
Thanks @native-api !
